tests/test_arbitrages.py

```diff
@@ -0,0 +1,160 @@
+from mev_inspect.arbitrage import get_arbitrages
+from mev_inspect.schemas.swaps import Swap
+from mev_inspect.swaps import (
+    UNISWAP_V2_PAIR_ABI_NAME,
+    UNISWAP_V3_POOL_ABI_NAME,
+)
+
+
+def test_two_pool_arbitrage(get_transaction_hashes, get_addresses):
+    block_number = 123
```

Comment:
Def a neat way to construct fixtures/tests, but can we use the same test files to plug in real tx hashes and see expected arb profit output?

Reply:
We could add some tests using real transfers by saving the JSON in the tests. I'll add that

Otherwise I did some manual tests for blocks with arbs I found people post about on twitter.